Userland/Libraries/LibJS/Runtime/TypedArrayPrototype.cpp

```diff
@@ -1452,4 +1453,36 @@ JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::map)
     return return_array;
 }
 
+// 23.2.3.28 %TypedArray%.prototype.toLocaleString ( [ reserved1 [ , reserved2 ] ] ), https://tc39.es/ecma262/#sec-%typedarray%.prototype.tolocalestring
+JS_DEFINE_NATIVE_FUNCTION(TypedArrayPrototype::to_locale_string)
+{
+    auto* typed_array = validate_typed_array(global_object);
+    if (!typed_array)
+        return {};
+
+    auto length = typed_array->array_length();
+
+    StringBuilder builder;
+    for (u32 k = 0; k < length; ++k) {
+        if (k > 0)
+            builder.append(','); // NOTE: Until we implement ECMA-402 (Intl) this is implementation specific.
+        auto value = typed_array->get(k);
+        if (vm.exception())
+            return {};
+        if (value.is_nullish())
+            continue;
+        auto* value_object = value.to_object(global_object);
+        if (!value_object)
+            return {};
+        auto locale_string_result = value_object->invoke(vm.names.toLocaleString.as_string());
```

Comment:
wait, does `invoke` not take a PropertyName?!

Reply:
it takes a StringOrSymbol weirdly, should i fix it in this PR?

Reply:
oh hmm. feel free to!